cognite/neat/rules/exporters/_rules2dms.py

```diff
@@ -40,8 +40,6 @@ class DMSExporter(CDFExporter[DMSSchema]):
             If set, only export components in the given spaces. Defaults to None which means all spaces.
         existing_handling (Literal["fail", "skip", "update", "force"], optional): How to handle existing components.
             Defaults to "update". See below for details.
-        standardize_casing(bool, optional): Whether to standardize the casing. This means PascalCase for external ID
```

Comment:
Agree that this should not be `NEAT` feature, we can suggest in best practices to use `PascalCase` for classes and `camelCase` for properties but that is all

Reply:
Yes absolutely.